### PR TITLE
Fix for issue #12 - added LICENSE.md file

### PR DESCRIPTION
**Pull request by deepanjali19**
Hey I have added the licence file. Please let me know if it needs any changes. 👍

### CODE REVIEW

File: src/LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright <2018> <Mera Gangapersaud>
```
Comment (SeanPrashad) [inline src/LICENCE.md]: Check out how this renders:

https://github.com/Mera-Gangapersaud/Creative-Collab/blob/651c55f23e0e3cbde7c334ff34afffd486e8b16b/src/LICENCE.md

You'll notice that Mera's name is missing! 🙀

We'll need to update this to `Copyright 2018 Mera Gangapersaud`

**Comment by SeanPrashad** [comment]
In addition, we'll need to update the name of the file from `LICENCE.md` to `LICENSE.md` - thank you!

File: src/LICENCE.md (hunk 1 above)
Comment (deepanjali19) [reply src/LICENCE.md]: I will work on that 👍🏻😊

**Comment by SeanPrashad** [comment]
You'll also need to perform a rebase - @humphd had walked us through it in class! But if you forget, it's no biggie ✌🏽

We'll need to do a few things:

1. Remove the `src/LICENCE.md` file and leave only `src/LICENSE.md` remaining:

    * `git rm src/LICENCE.md && git commit --amend --no-edit`

2. Move the `LICENSE.md` from `src` into the root directory:

    * `git mv src/LICENSE.md . && git commit --amend --no-edit`

3. Perform a git rebase to pull in the latest changes from `master` and add your own commits *afterwards*:

    * `git checkout master && git pull && git checkout - && git rebase master && git push -f`

4. You're done! We'll be notified that you've updated your PR with the above three steps. Let us know if anything is unclear

**Comment by deepanjali19** [comment]
Again a problem! I need @humphd for this rescue.

**Comment by SeanPrashad** [comment]
Please let us know how we can also help. Any error messages and/or screenshots would be great!

**Comment by deepanjali19** [comment]
Hey, I worked with @humphd and these test fails do not seem to make sense. I would appreciate some advise on how to fix this.

**Pull request merged**